chrome/skin/default/zotero/integration.css

```diff
@@ -216,8 +216,12 @@ span.zotero-bubble-input {
 	-moz-window-dragging: no-drag;
 }
 
-.citation-dialog.progress-meter[value] {
+.citation-dialog progress {
 	visibility: visible;
+	border-radius: 10px;
+	margin: 0;
+	flex: 1;
+	height: 10px;
```

Comment:
Should we also add `align-self: center;` so that the progress bar is in the center of the dialog, as opposed to being pushed into the upper-half?

<img width="799" alt="Screenshot 2025-01-29 at 10 17 16 AM" src="https://github.com/user-attachments/assets/126c556a-b00c-4511-aab4-cc7beaf0107d" />

Reply:
Not relevant since the dialog just has to (and will now) resize to only contain the progress bar.